api/views/user_views.py

```diff
@@ -204,6 +207,7 @@ def post(self, request):
         if hasattr(user, "instructor"):
             response["instructor"] = user.instructor.id
 
+        logger.info(f"User {user.id} logged in. Token: {token.key}")
```

Comment:
We shouldn't log secrets, so we shouldn't log the key here.

Reply:
Good catch, this was supposed to be removed after testing.